test/e2e/events_test.go

```diff
@@ -199,4 +199,17 @@ var _ = Describe("Podman events", func() {
 
 		wg.Wait()
 	})
+
+	It("podman events pod creation", func() {
+		SkipIfNotFedora()
+		stop := podmanTest.Podman([]string{"pod", "create", "--name", "foobarpod"})
```

Comment:
Shouldn't this have an `--infra=false` in it?

Reply:
yep, sorry